components/search-filters/index.tsx

```diff
@@ -26,17 +48,51 @@ const SearchFilters: React.FC<{
         addSaveClearButton
       >
         <label>Code postal ou numéro de département :</label>
-        <input
-          className="fr-input"
-          id="search-localisation-input"
-          autoComplete="off"
-          placeholder="ex: 35000"
+        <AsyncSelect
+          cacheOptions
+          defaultOptions
           name="cp_dep"
-          defaultValue={cp_dep}
+          noOptionsMessage={() =>
+            'Saisissez une ville ou un département pour rechercher son code.'
```

Comment:
This is a problem : we display the same message when API return no results and when nothing has been typed yet.